nixos/modules/config/console.nix

```diff
@@ -82,8 +82,7 @@ in
 
     packages = mkOption {
       type = types.listOf types.package;
-      default = with pkgs.kbdKeymaps; [ dvp neo ];
-      defaultText = "with pkgs.kbdKeymaps; [ dvp neo ]";
```

Comment:
Why did you remove them?

Reply:
as written in the pr description, they are included in the new version of kbd

you can see that the tests still works